docs/tutorials/1.5/build.md

```diff
@@ -0,0 +1,50 @@
+This is a small tutorial on how to build HORNET.
+
+**Note: This tutorial assumes that you are using Ubuntu. The setup and build process may differ on other OS.**
+
+### Preparations
+
+- Install go1.16:
+  ```bash
+     sudo add-apt-repository ppa:longsleep/golang-backports
```

Comment:
I would recommend to use the official golang.com page, with the wget command and the tar.... tbh :sweat_smile: